lib/acts_as_taggable_on_mongoid/models/concerns/tagging_validations.rb

```diff
@@ -10,8 +10,8 @@ module TaggingValidations
         included do
           validates :tag_name, presence: true
           validates :context, presence: true
-          validates :tag, presence: true
-          validates :taggable, presence: true
+          #validates :tag, presence: true
```

Comment:
And here?

Reply:
Rails 5 implicitly adds these validations with `belongs_to`.

I don't remember why I commented instead of just deleting.